### ISSUE TITLE
WebP rewrite rules from NGINX issue

### ISSUE THREAD

**Issue by markonikolic985** [wp-media/imagify-plugin#423]
It appears to be a problem with NGINX rewrite rules for WebP display.

From a user:

> I got the following error when running nginx -t : 

`- invalid parameter "final" in /var/www/go2roues.com/htdocs/conf/imagify.conf:12 -`

Current recommended NGINX rules by us:

```
location ~* ^(/.+)\.(jpg|jpeg|jpe|png|gif)$ {
    add_header Vary Accept;

    if ($http_accept ~* "webp"){
        set $imwebp A;
    }
    if (-f $request_filename.webp) {
        set $imwebp  "${imwebp}B";
    }
    if ($imwebp = AB) {
        rewrite ^(.*) $1.webp final;
    }
}
```

Meanwhile, suggesting to the user to remove `final` and see if that helps. The ticket can be found here - https://app.intercom.io/a/apps/cd6nxj3z/inbox/inbox/unassigned/conversations/23239228924

**Comment by aerd2** [comment]
Hello, all you have to do is to replace word "final" with "last" and it should work as intended.

**Comment by iCaspar** [comment]
Resolved on #488.
